mat/vector.go

```diff
@@ -482,3 +482,31 @@ func (v *VecDense) asGeneral() blas64.General {
 		Data:   v.mat.Data,
 	}
 }
+
+// ColViewOf reflects the column j of the RawMatrixer m, into the receiver
```

Comment:
This needs to have documentation explaining that the sizes of the values do not need to match. Or alliteratively make a test for `IsZero` and allow free assignment in that case and otherwise test for size.

Reply:
I think it should be that either the size matches or `IsZero`. This overwrites data and I'd like to do what we can to minimize that error. I'd lean toward forcing `IsZero`, but that's too burdensome with loops.